backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts

```diff
@@ -918,6 +918,14 @@ export class ExperimentAssignmentService {
   }
 
   public async dataLog(userDoc: RequestedExperimentUser, jsonLog: ILogInput[], logger: UpgradeLogger): Promise<Log[]> {
+    if (!userDoc) {
```

Comment:
This test will not filter out preview users, since you can /init them and they'll have a userDoc